data/templates/default.context

```diff
@@ -41,10 +40,12 @@ $endif$
 $if(pdfa)$
 % attempt to generate PDF/A
 \setupbackend
-  [format=PDF/A-1b:2005,
+  [format=PDF/A-$pdfa$,
    intent=sRGB IEC61966-2.1,
    profile=sRGB.icc]
 $endif$
+\setupbackend[export=yes]
```

Comment:
This just exports the document to XML.  I'm pretty sure that this is not needed to get tagging right, unless you want to embed the XML into the PDF.

Reply:
I tried removing it from the template, but then the Poppler `pdfinfo --struct-text` yields empty output.

Reply:
I also tested this on macOS using `context 2018.04.04`. Without `-M pdfa=true`, the output of `pdfinfo -struct-text` seems always correct:

```
Div
  Sect "section"
    Div
      H (block)
        "hi there"
    Div
      "how are things going today?"
```

But when setting the `pdfa` variable, the `\setupbackend[export=yes]` seems to be needed, otherwise I get:

```
Div
  Sect "section"
    Div
      H (block)
        "hithere"
    Div
      "howarethingsgoingtoday?"
```

btw. I'm curious, what is the syntax of that output? is it part of the pdf standard and implemented by context, or is it just how `pdfinfo` displays it?